en/docs/install-and-setup/setup/setting-up-databases/data-dictionary/user-management-related-tables.md

```
@@ -0,0 +1,474 @@
+# User Management Related Tables
+
+This section lists out all the user management related tables and their attributes in the WSO2 API Manager database.
+
+---
+
+### UM_ACCOUNT_MAPPING
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)